packages/bridge/README.md

```
@@ -1 +1,671 @@
+
+    // 동기 응답
+    return { success: true };
+  }}
+/>
+```
+
+## 문제 해결
+
+### 메시지가 전달되지 않아요
+
+**Handshake가 완료되었는지 확인하세요**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)